src/components/AssessmentView.jsx

```diff
@@ -64,16 +66,15 @@ const AssessmentView = ({
   isCommentEditId,
   setCommentText,
   setIsCommentEditId,
-  setIsMode
+  setIsMode,
+  handleRemoveFile,
+  isAssessmentLoading
 }) => {
   const { id: batchId } = useParams();
   const { token, user } = useAuth();
-  const [loading, setLoading] = useState(false);
-  const [isEditing, setIsEditing] = useState(false);
-  const [initialTitle, setInitialTitle] = useState("");
   const [studentScore, setStudentScore] = useState([]);
   const [toggleAssigneeWeightage, setToggleAssigneeWeightage] = useState(
-    type == "task" ? 0 : 1
+    type == "TASK" ? 0 : 1
```

Comment:
maintian this type of constants in another file.

Reply:
We Need disscussion